mentoring/meet-our-contributors.md

```diff
@@ -0,0 +1,39 @@
+# Meet Our Contributors - Ask Us Anything! 
+*This doc is a work in progress*
+
+When Slack seems like it’s going too fast, and you just need a quick answer from a human...
+
+Meet Our Contributors gives you a monthly one-hour opportunity to ask questions about our upstream community, watch interviews with our contributors, and participate in peer code reviews.
+
+## When:
+Every first Wednesday of the month at the following times -
+* 03:30pm UTC [calendar link](https://calendar.google.com/event?action=TEMPLATE&tmeid=MGw2OHM5bGRranJwZjI3OHN1MTk2Y2pjYjggcGFyaXNwaXR0bWFuQGdvb2dsZS5jb20&tmsrc=parispittman%40google.com)
+* 09:00pm UTC [calendar link](https://calendar.google.com/event?action=TEMPLATE&tmeid=N3M4dnZ0dGZwYTZzdXBuM3FndHRybGY0MTlfMjAxODAyMDdUMjEwMDAwWiBwYXJpc3BpdHRtYW5AZ29vZ2xlLmNvbQ&tmsrc=parispittman%40google.com&scp=ALL)
+
+Tune into the [Kubernetes YouTube Channel](https://www.youtube.com/c/KubernetesCommunity/live) to follow along with video and [#meet-our-contributors](https://kubernetes.slack.com/messages/meet-our-contributors) on Slack for questions and discourse. 
+
+## What’s on-topic: 
+* How our contributors got started with k8s
+* Advice for getting attention on your PR
+* GitHub tooling and automation
+* Your first commit
+* kubernetes/community
+* Testing 
+
+## What’s off-topic:
+* End-user questions (Check out [#office-hours](https://kubernetes.slack.com/messages/office-hours) on slack and details [here](events/officehours.md))
+
+## Submitting a question:
+Day of on twitter - use #k8smoc   
```

Comment:
I would suggest expanding the call to action here to be something like "use the hashtag #k8smoc". You could even link to https://twitter.com/hashtag/k8smoc which would allow someone to follow the conversation, or tweet with the hashtag right from that page

Reply:
I would also make this a bulleted list too